### PR TITLE
SW_VERS should be lowercase

### PR DESCRIPTION
**Pull request by ohayon**

### CODE REVIEW

**Review by mpirri — APPROVED** [review]
:shipit:

**Comment by fastlane-bot-helper** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please provide a changelog summary in the Pull Request description @ohayon</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: fastlane_core/lib/fastlane_core/analytics/analytics_session.rb [hunk 1]
```diff
@@ -191,7 +191,7 @@ def operating_system_version
       os = self.operating_system
       case os
       when "macOS"
-        return `SW_VERS -productVersion`.strip
+        system('sw_vers') ? return `sw_vers -productVersion`.strip : ''
```
Comment (mpirri) [inline fastlane_core/lib/fastlane_core/analytics/analytics_session.rb]: Should we use a string like `unknown` like we do with unknown p-hashes?  (easier to query against, maybe?)

**Comment by mpirri** [comment]
:shipit: x2!

**Comment by mpirri** [comment]
good catch!!!!!!!  :shipit: x3

**Comment by casz** [comment]
👎  @ohayon 
Deployed internally and it's leaking sw_vers everywhere 💥 

Just running `fastlane` shows sw_vers info...
### Captured Output

Command Used: `fastlane`
<details><summary>Output/Log</summary>

```
                                                                                                                                                                                                    
[13:08:52]: Get started using a Gemfile for fastlane https://docs.fastlane.tools/getting-started/ios/setup/#use-a-gemfile
ProductName:	Mac OS X
ProductVersion:	10.13
BuildVersion:	17A405
[13:08:54]: Welcome to fastlane! Here's what your app is setup to do:
+--------+-----------------+------------------------------+
|                 Available lanes to run                  |
+--------+-----------------+------------------------------+
| Number | Lane Name       | Description                  |
+--------+-----------------+------------------------------+
| 1      | devices         |                              |
| 0      | cancel          | No selection, exit fastlane! |
+--------+-----------------+------------------------------+
[13:08:54]: Which number would you like run?
0

[!] Run `fastlane` the next time you need to build, test or release your app 🚀


```

</details>

<details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                         | Value                                         |
| --------------------------- | --------------------------------------------- |
| OS                          | 10.13                                         |
| Ruby                        | 2.4.2                                         |
| Bundler?                    | false                                         |
| Git                         | git version 2.14.2                            |
| Installation Source         | /usr/local/bin/fastlane                       |
| Host                        | Mac OS X 10.13 (17A405)                       |
| Ruby Lib Dir                | /usr/local/Cellar/ruby/2.4.2/lib              |
| OpenSSL Version             | OpenSSL 1.0.2l  25 May 2017                   |
| Is contained                | false                                         |
| Is homebrew                 | false                                         |
| Is installed via Fabric.app | false                                         |
| Xcode Path                  | /Applications/Xcode-9.app/Contents/Developer/ |
| Xcode Version               | 9.0                                           |


### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_US.UTF-8 | ✅ |
| LC_ALL   |             |   |
| LANGUAGE |             |   |


### fastlane files:

__snip__

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.62.1  | ✅ Up-To-Date  |


### Loaded fastlane plugins:

**No plugins Loaded**


<details><summary><b>Loaded gems</b></summary>

| Gem                       | Version      |
| ------------------------- | ------------ |
| did_you_mean              | 1.1.0        |
| slack-notifier            | 1.5.1        |
| CFPropertyList            | 2.3.5        |
| claide                    | 1.0.2        |
| colored2                  | 3.1.2        |
| nanaimo                   | 0.2.3        |
| xcodeproj                 | 1.5.2        |
| rouge                     | 2.0.7        |
| xcpretty                  | 0.2.8        |
| terminal-notifier         | 1.8.0        |
| unicode-display_width     | 1.3.0        |
| terminal-table            | 1.8.0        |
| plist                     | 3.3.0        |
| multipart-post            | 2.0.0        |
| word_wrap                 | 1.0.0        |
| public_suffix             | 2.0.5        |
| tty-screen                | 0.5.0        |
| babosa                    | 1.0.2        |
| colored                   | 1.2          |
| highline                  | 1.7.8        |
| commander-fastlane        | 4.4.5        |
| excon                     | 0.59.0       |
| faraday                   | 0.13.1       |
| unf_ext                   | 0.0.7.4      |
| unf                       | 0.1.4        |
| domain_name               | 0.5.20170404 |
| http-cookie               | 1.0.3        |
| faraday-cookie_jar        | 0.0.6        |
| fastimage                 | 2.1.0        |
| gh_inspector              | 1.0.3        |
| mini_magick               | 4.5.1        |
| multi_json                | 1.12.2       |
| multi_xml                 | 0.6.0        |
| rubyzip                   | 1.2.1        |
| security                  | 0.1.3        |
| xcpretty-travis-formatter | 0.0.4        |
| dotenv                    | 2.2.1        |
| bundler                   | 1.14.6       |
| faraday_middleware        | 0.12.2       |
| uber                      | 0.1.0        |
| declarative               | 0.0.10       |
| declarative-option        | 0.1.0        |
| representable             | 3.0.4        |
| retriable                 | 3.1.1        |
| addressable               | 2.5.2        |
| mime-types-data           | 3.2016.0521  |
| mime-types                | 3.1          |
| httpclient                | 2.8.3        |
| google-api-client         | 0.13.6       |
| json                      | 2.1.0        |
| io-console                | 0.4.6        |
| openssl                   | 2.0.5        |
</details>


*generated on:* **2017-10-18**
</details>

**Comment by casz** [comment]
@ohayon it's the `system('sw_vers')` that is leaking

**Comment by casz** [comment]
Fixed in #10630

**Pull request merged**